index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Mohasin Haque">
+    <meta name="description"
+        content="FAST UI a CSS library with pre-defined classes for quick creation of your projects. Now build projects more faster by just in single click by integrating this FAST UI in your code.">
+    <link rel="icon" href="https://img.icons8.com/ios-glyphs/50/000000/fast-track.png">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.7.2/font/bootstrap-icons.css">
+    <link rel="stylesheet" href="./styles/main.css">
+    <title>FAST UI</title>
+</head>
+
+<body>
+    <nav class="navbar">
+        <div class="left-nav">
+            <a href="#" class="nav nav-text">FAST UI</a>
+            <span class="small-text">V 1.0</span>
+        </div>
+        <div class="right-nav">
+            <ul class="list">
+                <li><a href="./index.html" class="nav right right-link">Home</a></li>
+                <li><a href="./pages/documentation.html" class="nav right right-link">Documentation</a></li>
+                <li><a href="./pages/installation.html" class="nav right right-link">Installation</a></li>
+            </ul>
+        </div>
+    </nav>
+    <section class="section section-main">
+        <div class="container">
+            <h3 class="heading">Integrate FAST to Make FAST</h3>
+        </div>
+        <p class="about">Welcome to FAST UI Component Library. FAST UI is a CSS library with pre-defined classes for
+            quick creation of your projects. Now build projects more faster by just doing single click by integrating
+            this FAST UI in your code.</p>
+        <div class="btn-container">
+            <a class="btn btn-primary" href="./pages/installation.html">Installation</a>
+            <a class="btn btn-secondary" href="./pages/documentation.html">Documentation</a>
```

Comment:
You can added indentation to the codebase.

Reply:
Done!! Formatted the code again and also used indentation rainbow extension for indentation.